llpc/util/llpcElfReader.cpp

```diff
@@ -321,7 +321,7 @@ void ElfReader<Elf>::GetSymbolsBySectionIndex(
                 secSymbols.push_back(symbol);
             }
         }
-
+        
```

Comment:
Please configure your editor to automatically remove whitespace at the end of a line.

Reply:
Done.